lib/hanami/router.rb

```diff
@@ -874,8 +878,11 @@ def prefixed_path(path)
 
     # @since 2.0.0
     # @api private
```

Comment:
Let's reset this to `@since x.x.x` since we've changed the method name.

TBH, I'm in favour of dropping the `@since` on anything that's marked `@api private`. It just adds unnecessary busywork for us. This is a change for another time, though :)

Reply:
Oh, I also notice that this is actually `private` in terms of visibility too. Yeah, I'd definitely support all the `@since` and `@api` tags being removed from these methods.

Reply:
I've reset it as per your suggestion and created an issue on the mother ship repo for dropping all `@since` for private APIs at some point: https://github.com/hanami/hanami/issues/1516